.github/CODEOWNERS

```diff
@@ -15,6 +15,9 @@
 # Module: odata-service-writer
 /packages/odata-service-writer/ @SAP/ux-tools-odata-service-writer
 
+# Module: ui5-components
+/packages/ui5-components/ @SAP/ux-tools-ui5-components
```

Comment:
These are not ui5 components but UI components, correct?

Reply:
yes I have adapted. 

@tobiasqueck One more thing in the old PR you had mentioned to have [same version](https://github.com/SAP/open-ux-tools/pull/661#discussion_r964801884) for typescript as of root but when i maintain in such a way i get following error during pnpm install
![image](https://user-images.githubusercontent.com/103586139/189604100-d36743c5-6c9f-4ca3-9d70-15b36787da7a.png)

In `ui-components` package we are using `'^4.6.3'` and in root is `'4.0.8'` so when updating to `'4.0.8'` in `ui-components` we get the peer deps error in `pnpm install`

Reply:
updated in https://github.com/SAP/open-ux-tools/pull/671/commits/04e0aaf51e0f0c1f4fe7fba3f5b06308bfd91df1